docs/kpod-attach.1.md

```diff
@@ -0,0 +1,30 @@
+% kpod(1) kpod-attach - See the output of pid 1 of a container or enter the container
+% Dan Walsh
+# kpod-attach "1" "September 2017" "kpod"
+
+## NAME
+kpod-attach - Attach to a running container
+
+## Description
+
+We chose not to implement the `attach` feature in `kpod` even though the upstream Docker
+project has it. The upstream project has had lots of issues with attaching to running processes.
+
+**Reasons to attach to the primary PID of a container:**
+
+
+
+1) Viewing the output stream of the container
+
+  We recommend that you use `kpod logs` to see the output
+
+  `kpod logs CONTAINERID`
+   
+2) Executing commands inside of the container
```

Comment:
I'd suggest making exec be number one and logs be number two in this list.  I think most will want to attach and it's alphabetical.